### ISSUE TITLE
Informative error messages on catalogue interloper KeyErrors

### ISSUE THREAD

**Issue by edoaltamura** [SWIFTSIM/velociraptor-python#101]
We could raise errors with a better-documented message when encountering interloper-related errors, such as `KeyError: 'interloper'`. For instance, version 0.10.0 cannot load interlopers in zoom simulations, but version 0.15.8 supports them - additional info can be printed such as `"Interlopers found in <*.properties> do not match expectations. You can try updating velociraptor by running 'pip install velociraptor --upgrade'"`.

**Comment by MatthieuSchaller** [comment]
Right, but improving the error message won't help with people running an old version.
